README.md

```diff
@@ -70,8 +76,19 @@ The bootstrap.sh can be invoked multiple times but it has a status control to en
 of the libraries is loaded only once. This is to prevent potential issues with collections 
 that are not supposed to be loaded many times.
 
-An example of this is the _container_engine_ library, where the selected container engine
-is **set only once the first command using the library helper function `container_engine_cmd` is used**.
+An example of this is the _container_ library, where the selected container engine
+is **set only once the first command using the library helper function `cicd_tools::container::cmd` is used**.
+
+Each of the libraries will export to the shell sourcing the bootstrap script the helper functions.
```

Comment:
The wording here is confusing to me :) Do you mean something like this?:
```
Each of the libraries will be exported to the shell when sourcing the bootstrap script
```

Reply:
I did, definitely a typo, probably from when I introduced the latest review (after separating `main` and `bootstrap` scripts. Thank you for spotting this.